test/support/application_fixture_helper.rb

```diff
@@ -58,7 +58,7 @@ def copy_dir(path)
   end
 
   def fixture_path
-    File.join(TEMP_FIXTURE_DIR, "#{name}-#{Time.now.strftime("%Y%m%d")}")
+    File.join(TEMP_FIXTURE_DIR, "#{name}-#{Time.now.strftime('%Y%m%d')}")
```

Comment:
Can you update the first commit to fix this rubocop failure. I think this is appropriate since you are introducing the failure in that commit and it is not yet on main.

Reply:
Sure! I'll roll that in when I update the commits with final reviewer suggestions.